chromium_src/third_party/blink/renderer/modules/plugins/navigator_plugins.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_NAVIGATOR_PLUGINS_SHOULD_RETURN_FIXED_PLUGIN_DATA \
+  return false;                                                 \
```

Comment:
when does this actually happen? Can't we force it to always return `false` by setting `allow_non_empty_navigator_plugins` to `true` in `BraveContentBrowserClient::OverrideWebkitPrefs` https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/chrome_content_browser_client.cc;l=3415?q=web_prefs-%3Eallow_non_empty_navigator_plugins&ss=chromium%2Fchromium%2Fsrc

Reply:
Changed in 07bc14d